packages/dapp-connector-client/package.json

```diff
@@ -52,9 +52,9 @@
   },
   "dependencies": {
     "@cardano-sdk/core": "workspace:~",
+    "@cardano-sdk/key-management": "workspace:~",
```

Comment:
What is the fix here? :thinking:

Reply:
None, but this came from one of the merges, I'll revert.